Gruntfile.coffee

```diff
@@ -128,21 +128,17 @@ module.exports = (grunt) ->
           dest: '<%= webAssetsPath %>/images/'
         ]
 
-    # Webfont configuration
-    webfont:
-      all:
-        src:  [
-          'src/**/Resources/assets/icons/*'
+    # Grunticon configuration
```

Comment:
Ik zou dit ook omschrijvingen in de description van de PR, zodat het duidelijk is wat er aangepast is, en hoe je kan upgraden. Want bundles die nog met aparte icons zouden werken gaan nu niet meer werken